### DISCUSSION TITLE
Option: Always reserve stage manager space

### DISCUSSION THREAD

**Discussion by sbscan** [rxhanson/Rectangle#1516] [Ideas]
Rectangle respects Stage Manager, but this behavior is not very convenient. At the very least, it should be optional.
When there are no apps in Stage Manager, it occupies the whole screen. This leads to:

Inability to use Stage Manager smoothly due to lack of space.
Windows being arranged in different positions depending on the situation.

**Comment by rxhanson** [comment]
Thanks for the feedback. This isn’t a change I have plans to add in myself, but see no reason to turn down a good pull request that adds this as a configuration set via Terminal command (not in the UI).

**Comment by anthonysmillie** [comment]
That's a shame. Came to the repo looking to see if someone had already reported this. Would love this functionality myself, both for rectangle and for multitouch.

**Comment by rxhanson** [comment]
Although not identical to the original ask, there is the ability to set a screen edge gap via Terminal command:

https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#setting-gaps-at-the-screen-edges

If stage is on the left:
> ```
> defaults write com.knollsoft.Rectangle screenEdgeGapLeft -int 190
> ```
If stage is on the right:
>```
> defaults write com.knollsoft.Rectangle screenEdgeGapRight -int 190
> ```

Restart the app after executing those. You would probably want to set the stage manager size in Rectangle to 0.

In Multitouch, you would swap out `com.knollsoft.Rectangle` with `com.brassmonkery.Multitouch`
